src/app/shared-services/informed-consent/informed-consent.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core'
+import { Observable, of } from 'rxjs'
+import { InformedConsentDefaults } from './informed-constent.fixtures'
+
+@Injectable({
+  providedIn: 'root',
+})
+export class InformedConsentService {
+  constructor() {}
+
+  getInformedConsent(key: string): Observable<InformedConsentDefaults> {
```

Comment:
Maybe it is more elegant to define a static key here and use that instead of using a parameter (which seems to always be 'consent' either way?)